activejob/lib/active_job/exceptions.rb

```diff
@@ -126,17 +126,20 @@ def retry_job(options = {})
     end
 
     private
-      def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: nil)
-        jitter ||= self.class.retry_jitter
+      JITTER_DEFAULT = Object.new
```

Comment:
This is not really private if you don't call `private_constant`, so it is better to move it to the public section or keep it here and also call `private_constant`.

Reply:
Good catch. Thanks. I've made it private in https://github.com/rails/rails/pull/38003/commits/98439f3d5c510cd413fac4e71b8e460343e30cff